led_strip/CMakeLists.txt

```diff
@@ -2,8 +2,10 @@ include($ENV{IDF_PATH}/tools/cmake/version.cmake)
 
 set(srcs "src/led_strip_api.c")
 
-if(CONFIG_SOC_RMT_SUPPORTED)
```

Comment:
why remove this? FYI, we have some chip that doesn't have the RMT peripheral, e.g. ESSP32C2

Reply:
I am sorry, I thought, that this constant is not available in IDF4.4 - I'll fix it.

Reply:
Fixed

Reply:
> I thought, that this constant is not available in IDF4.4 

Sorry @espzav You're right, soc caps are injected into the Kconfig system starting from esp-idf v5.0. I'm afraid this statement will be false when we build it with esp-idf v4.4

Reply:
You are right @suda-morris, I was tested, but I didn't remove whole build folder before and I though, it is working... I changed the condition only for IDF5 and higher.